assets/js/modules/analytics-4/components/widgets/LeastEngagingPagesWidget.js

```diff
@@ -137,13 +142,15 @@ function LeastEngagingPagesWidget( props ) {
 		return pageViewsReportError || reportError || undefined;
 	} );
 
-	const titles = useInViewSelect( ( select ) =>
-		! error
-			? select( MODULES_ANALYTICS_4 ).getPageTitles(
-					report,
-					reportOptions
-			  )
-			: undefined
+	const titles = useInViewSelect(
+		( select ) =>
+			! error
+				? select( MODULES_ANALYTICS_4 ).getPageTitles(
+						report,
+						reportOptions
+				  )
+				: undefined,
+		[ report, reportOptions ]
```

Comment:
We need to add `error` as dependency as well.

Reply:
Great spot, thanks, I missed that one